plugins/pipelines/helper.js

```diff
@@ -26,6 +26,15 @@ const formatCheckoutUrl = (checkoutUrl) => {
     return result;
 };
 
+/**
+ * Get rid of leading/trailing slashes in rootDir, return empty string as default
+ * @method sanitizeRootDir
+ * @param  {String}     rootDir     Root directory (ex: /src/component/app/ or /)
+ * @return {String}                 Root dir with no leading/trailing slashes
+ */
+const sanitizeRootDir = (rootDir = '') => rootDir.replace(/^\/+|\/+$/g, '');
```

Comment:
can we also validate if it's an actual path ?

Reply:
you mean check if it exists in the repo? that's going to be hard, need to make git calls

Reply:
Not checking if it actually exits in repo, just patterns for valid file path. Eg: /x/y/z or ./s or s/y

Reply:
I think that would be hard to do correctly. Here are some of the top results based on a quick search online:
- https://stackoverflow.com/questions/16231210/file-path-validation-in-javascript
- https://www.codeproject.com/Tips/216238/Regular-Expression-to-Validate-File-Path-and-Exten
- https://www.npmjs.com/package/is-valid-path (this package was last published 4 years ago)

If you really want it, I can try to adapt one of those regex patterns.

Reply:
we need to look into only valid unix file paths.  

https://stackoverflow.com/questions/28989672/regex-to-tell-if-a-string-contains-a-linux-file-path-or-if-a-linux-file-path-as
https://stackoverflow.com/questions/55069650/regex-pattern-to-validate-linux-folder-path

Reply:
added